examples/language_model/gpt-3/deploy/python/inference.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import tempfile
+from functools import partial
+
+import numpy as np
+import paddle
+import paddle.distributed.fleet as fleet
+from paddle import inference
+from paddlenlp.data import Stack, Tuple, Pad
+from paddlenlp.transformers import GPTForGreedyGeneration, GPTChineseTokenizer, GPTTokenizer
```

Comment:
1. remove importing in line 24: 「from paddlenlp.data import Stack, Tuple, Pad」
2. GPTForGreedyGeneration is not used

Reply:
Done